aspnetcore/client-side/spa/intro/mermaid/sequenceDiagram.txt

```diff
@@ -0,0 +1,35 @@
+```mermaid
+sequenceDiagram
+participant Browser
+participant Proxy
+participant Server
+Browser->>Server: GET /
+alt Proxy is running
```

Comment:
@peterblazejewicz what's better than `alt` and `par` in the Mermaid image below?
alt -> Proxy?